cookbooks/bcpc-hadoop/recipes/oozie.rb

```diff
@@ -106,23 +106,14 @@
   #code "#{OOZIE_LIB_PATH}/bin/oozie-setup.sh sharelib update -fs #{HDFS_URL}"
   code "#{OOZIE_CLIENT_PATH}/bin/oozie-setup.sh sharelib upgrade -fs #{HDFS_URL}"
   user "oozie"
-  not_if "sudo -u hdfs hdfs dfs -test -d #{HDFS_URL}/user/oozie/share/lib", :user => "hdfs"
-  only_if "echo 'test'|sudo -u hdfs hdfs dfs -copyFromLocal - /tmp/oozie-test"
-  notifies :run,"bash[delete-oozie-temp-file]",:immediately
+  not_if "hdfs dfs -test -d #{HDFS_URL}/user/oozie/share/lib", :user => "hdfs"
+  only_if "echo 'test'| hdfs dfs -copyFromLocal - /tmp/oozie-test && hdfs dfs -rm -skipTrash /tmp/oozie-test", :user => "hdfs"
   #not_if { require 'time'
   # hdfs_mtime=`hdfs dfs -stat #{share_dir_url}`.strip
   # Time.parse("#{hdfs_mtime} UTC") >
   # File.mtime("#{OOZIE_CLIENT_PATH}/oozie-sharelib.tar.gz") }
```

Comment:
Per @cbaenziger, these need to be uncommented again so sharelib is updated on hdfs. The current change will only make sure sharelib dir is created once on hdfs, it won't update if the on-disk sharelib has changed